app/grants/migrations/0017_matchpledge_end_date.py

```diff
@@ -14,6 +14,6 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='matchpledge',
             name='end_date',
-            field=models.DateTimeField(default=grants.models.next_month),
+            field=models.DateTimeField(default=grants.models.match_pledge.next_month),
```

Comment:
Hmm I'd say let's not touch older migration files cause it's already been run on production

I'm good adding it as a try-catch